configure.ac

```diff
@@ -2332,6 +2336,7 @@ echo "  JSON support                : $with_jsonc"
 echo " Build options:"
 echo "  Generate manual pages       : ${enable_manpages:=no}"
 echo "  Install manual pages        : ${enable_manpages_install:=no}"
+echo "  Example modules             : ${enable_example_modules:=yes}"
```

Comment:
Sorry, I just noticed:
Can we move this under the modules section?

Reply:
done